src/Control/Email/Swift/MailTransport.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * This file was copied in from swiftmailer/swiftmailer v5.4.12 after it was removed from switftmailer v6
+ * It has been slightly modified to meet phpcs standards and initialise Swift_DependencyContainer
+ */
+
+/*
+ * This file is part of SwiftMailer.
+ * (c) 2004-2009 Chris Corbyn
+ *
+ * For the full copyright and license information, please view the LICENSE file (MIT)
+ * https://github.com/swiftmailer/swiftmailer/blob/181b89f18a90f8925ef805f950d47a7190e9b950/LICENSE
+ */
+
+/**
+ * Sends Messages using the mail() function.
+ *
+ * @author Chris Corbyn
+ *
+ * at deprecated since 5.4.5 (to be removed in 6.0)
+ *
+ * @internal
+ */
+// @codingStandardsIgnoreStart
+// ignore missing namespace
+class Swift_MailTransport extends Swift_Transport_MailTransport
```

Comment:
Does not seem like it's breaking it?  `class_exists('Swift_MailTransport')` in PageController::init() returns true. No issues with ?flush, email get sent

Reply:
It doesn't break because of the `ClassManifest` voodoo magic that finds all the classes on flush.

But using the composer autoloader is better in the long run:
- it's more standard compliant with the rest of the PHP ecosystem,
- it's meant to be a tiny bit faster.

That's why in our doc [we recommend modules set up autoloading](https://docs.silverstripe.org/en/4/developer_guides/extending/modules/#modules) even if, strictly speaking, it's not need to.
> Like with any Composer package, we recommend declaring your PHP classes through PSR autoloading. Silverstripe CMS will automatically discover templates and configuration settings within your module when you next flush your cache.

The options here are either:
- Keep the files where they are and rename them to be part of the `SilverStripe` namespace
- Move the file to `thirdparty/swift` and add a PSR-0 autoloading entry to our composer file. e.g.: `"autoload": {"psr-0": { "Swift_": "thirdparty/swift" }},`

Reply:
I've moved this to the thirdparty library and autloading it via 'classmap' which can be used for non-namespaced classes

Reply:
I think swiftmailer implicitly follows the PSR-0 standard. But classmap works as well. I tested this locally with this composer file. Confirmed that there's autoload entries being generated for all our bastard swiftmailer classes.

```json
{
    "name": "max/composer-test",
    "autoload": {
        "psr-4": {
            "Max\\ComposerTest\\": "src/"
        }
    },
    "authors": [
        {
            "name": "Maxime Rainville",
            "email": "maxime@silverstripe.com"
        }
    ],
    "require": {
    	"silverstripe/framework": "dev-pulls/4/swiftmailer-v6-support as 4.x-dev"
    },
    "repositories": [
        {
            "type": "vcs",
            "url": "git@github.com:creative-commoners/silverstripe-framework.git"
        }
    ],
    "license": "BSD-3-Clause",
    "description": "boom"
}
```